api/search.py

```diff
@@ -29,6 +29,14 @@ def _execute(self, word, fieldname):
         results = search.scan()
         return results
 
+    def subquery(self, word):
+        """
+        Create a query for a single word of the phrase entered.
+        """
+        if "*" in word:
+            return Q("intervals",  cf={ "wildcard": { "pattern": word } })
```

Comment:
what does the intervals part mean?

Reply:
That's a great question, I wish I had documented it...